python/health-check/src/health_check/main.py

```diff
@@ -60,7 +62,7 @@ def cli(ctx: click.Context, supportconfig_path: str, verbose: bool):
     callback=utils.validate_date,
 )
 @click.pass_context
-def run(ctx: click.Context, from_datetime: str, to_datetime: str, since: int):
+def start(ctx: click.Context, from_datetime: str, to_datetime: str, since: int):
```

Comment:
Does this imply change in the docs? I.e. right now, we say to run `health-check run ...` - do we have to change it to `health-check start ...`, or is it just `health-check ...`? Either way, we'll probably need to modify the readme, right?

Reply:
That's true, we need to change the README and wiki documentation.

I'll include the documentation changes and some of your suggestions in a follow-up PR.